en/En.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class En {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        int sum = 0;
+        int minValue = Integer.MAX_VALUE;
+        int maxValue = Integer.MIN_VALUE;
+
+        for (int i=0; i<5; i++) {
```

Comment:
I don't think it is correct with this, I mean there should be one more loop, and some more operations.